packages/contracts-bedrock/snapshots/storageLayout/L1ERC721Bridge.json

```diff
@@ -1,38 +1,24 @@
 [
   {
-    "bytes": "1",
-    "label": "_initialized",
```

Comment:
L1ERC721Bridge is the only contract using v5 Initializable, so the `_initialized` slot no longer appears in this layout.